osmchadjango/changeset/management/commands/generate_harmful_csv.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand
+from ...models import Changeset, UserWhitelist, SuspicionReasons, SuspiciousFeature
+from ...filters import ChangesetFilter
+from djqscsv import write_csv
+
+class Command(BaseCommand):
+    help = 'Generate a CSV of all the harmful changesets from the database.'
+
+    def handle(self, *args, **options):
+        qset = Changeset.objects.filter(harmful=True).select_related('user_detail')
+        with open('harmful_changesets.csv', 'w') as csv_file:
```

Comment:
@geohacker could we place take output filename as a parameter or so?
Otherwise, this looks great to me!